schemas/cluster_version_mapping.schema.json

```diff
@@ -34,7 +33,7 @@
         "type": "string"
       },
       {
-        "description": "The file name of a pre-generated remote configuration that the Insights Operator Conditional Gathering Service v2 API will return for versions in the interval represented by the pair.",
+        "description": "A path to a remote configuration file that the Insights Operator Conditional Gathering Service v2 API will return for versions in the interval represented by the pair. The path must be relative to the location of the cluster_version_mapping.json file and must not point outside of its parent directory.",
```

Comment:
Do we need, if not already there, an assertion and a test that there will be an error if the location points outside the parent directory?

Reply:
You don't need PR descriptions, do you? :wink: 

> I still need to add a check to make sure the relative paths do not point outside of the parent directory of the file. Tests in tests/source_data_validation might deserve more changes, too. I will do that after this PR.

I'm working on it right now.

Reply:
:) By the time I got to the fifth commit I already forgot what was on the description. Sorry, I have a small memory buffer